packages/pytest-simcore/src/pytest_simcore/helpers/utils_assert.py

```diff
@@ -37,7 +40,15 @@ async def assert_status(
         if expected_msg:
             assert expected_msg in data["message"]
 
-    return data, error
+    return_value = (
+        data,
+        error,
+    )
+    if include_meta:
+        return_value += (json_response.get("_meta"),)
+    if include_links:
+        return_value += (json_response.get("_links"),)
+    return return_value
```

Comment:
It would be a nice touch, yes. but it would require changing a lot of test code, i guess that is why @sanderegg  adopted this approach. An alternative that might comply with both would be to create a separate wrap function and user there named tuples

```python
from collection import namedtuples

PageBody = namedtuples("PageBody", "data error _meta _links")
async assert_status_for_pagination(include_meta: bool, include_links: bool, *args, **kwargs):
     data, error = await assert_status(*args, **kwargs
     ...
```